modifications/NpntProcess.qml

```diff
@@ -30,26 +25,23 @@ import QGroundControl 1.0
             anchors.fill:parent
             z:2
         }
-
         Column{
             spacing: rect1.height/10
             anchors.centerIn: rect1
-
            //START NPNT CHECKLIST
             Column{
-            Text {
-                id: init
-                text: qsTr("STARTING INIT PROCESS..")
-                font.bold: true
-                font.pixelSize:(rect1.width/25 + rect1.height/25 )/2
-            }
+                Text {
+                    id: init
+                    text: qsTr("STARTING INIT PROCESS..")
+                    font.bold: true
+                    font.pixelSize:(rect1.width/25 + rect1.height/25 )/2
+                }
             }
-
             Column{
                     CheckBox {
                         id: c1
                         checked:myrect.hardwareConnected
-                        text: qsTr("Hardware Connected")
+                        text: qsTr("Connecting to the Flight Controller")
```

Comment:
Connecting or connected?

Reply:
Both are fine, the status is represented by a checkbox.  The text was given by @not-vibhu.
Currently changing it to "connected".  Can be reverted if needed